src/getgb2r.f

```diff
@@ -1,197 +1,159 @@
+C>    @file
+C>    @brief This subroutine read and unpack sections 6 and 7 from ah
+C>    grib2 message.
+C>    @author Stephen Gilbert @date 2002-01-11
+C>
+
+C>    This subroutine read and unpack sections 6 and 7 from a grib2
+C>    message. It assumes that the "metadata" for this field already
+C>    exists in derived type gribfield. Specifically, it requires
+C>    gfld\%ibmap, gfld\%ngrdpts, gfld\%idrtnum, gfld\%idrtmpl, and
+C>    gfld\%ndpts. It decoded information for the selected grib field and
+C>    returned it in a derived type variable, gfld. gfld is of type
+C>    gribfield, which is defined in module grib_mod, so users of this
+C>    routine will need to include the line "use grib_mod" in their
+C>    calling routine. Each component of the gribfield type is described
+C>    in the output argument list section below.
+C>
+C>    @param[in] LUGB integer unit of the unblocked grib data file.
+C>    @param[in] CINDEX index record of the grib field (see docblock of
+C>    subroutine ixgb2 for description of an index record.)
```

Comment:
Where function or subroutine names are used, include parens:

`C>    subroutine ixgb2() for description of an index record.)
`